script.js

```diff
@@ -24,4 +21,80 @@ function episodeCode(season, number) {
   let code = `S${s}E${num}`;
   return code;
 }
-window.onload = setup;
+
+const searchBox = document.getElementById("search");
+searchBox.addEventListener("input", handleSearchInput);
+function handleSearchInput(event) {
+  state.searchTerm = event.target.value;
```

Comment:
What will happen if user input is case-sensitive or case-insensitive? For example will I still get a result if I write like following words:
`Winter, WINTER, winter`?

Reply:
Remember to remove the console.log